habitat/core/__init__.py

```diff
@@ -3,3 +3,14 @@
 # Copyright (c) Facebook, Inc. and its affiliates.
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
+import enum
+
+
+class CHANGE_TASK_BEHAVIOUR(enum.IntEnum):
```

Comment:
Please move these to `env.py` and then import them from there here.  We haven't been defining things in `__init__.py` files so this would match that.